### DISCUSSION TITLE
OSD info duration

### DISCUSSION THREAD

**Discussion by buzzcut121** [smplayer-dev/smplayer#961] [Q&A] [answered]
Hi, I am trying to make part of the OSD remain on screen longer. By OSD I mean what's on the attached picture (time elapsed/total time/%).  

I have added --osd-duration=2000 to the mpv options but the only thing that is affected by varying the numeral is the progress bar. The elapsed time info duration does not change no matter what I do. Is there an option I am overlooking or one I need to add to the options list (assuming change is possible)? Thanks
![IMG_20240508_142929](https://github.com/smplayer-dev/smplayer/assets/93726880/57660c2e-d142-4804-b260-a758a71e8930)

**Comment by smplayer-dev** [comment, marked as answer]
Some OSD texts are displayed by smplayer and I'm afraid the duration is hard coded, so I can't be changed.
